src/js/messaging/containers/Thread.jsx

```diff
@@ -1,55 +1,62 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
-import { fetchThread } from '../actions/messages';
+import { fetchThread, setVisibleDetails } from '../actions/messages';
 import Message from '../components/Message';
+import NoticeBox from '../components/NoticeBox';
 
 class Thread extends React.Component {
   componentWillMount() {
```

Comment:
In light of what we learned with #3066/#3069, should this be `componentDidMount`?